sdcm/utils/docker_remote.py

```diff
@@ -102,7 +106,9 @@ def get_log(self):
         return self.node.remoter.run(f"{self.sudo_needed} docker logs {self.docker_id}").stdout.strip()
 
     def run(self, cmd, *args, **kwargs):
-        return self.node.remoter.run(f'{self.sudo_needed} docker exec {self.docker_id} /bin/sh -c {shlex.quote(cmd)}', *args, **kwargs)
+        if self.is_shell_supported:
+            return self.node.remoter.run(f'{self.sudo_needed} docker exec {self.docker_id} /bin/sh -c {shlex.quote(cmd)}', *args, **kwargs)
+        return self.node.remoter.run(f'{self.sudo_needed} docker exec {self.docker_id} {cmd}', *args, **kwargs)
```

Comment:
I think that something in SCT may fail without `shlex.quote` usage in this case.

Reply:
fixed